.github/workflows/release.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           git-ref: ${{ github.ref }}
 
       - name: download artifacts
-        uses: actions/download-artifact@158ca71f7c614ae705e79f25522ef4658df18253 # v2.0.9
+        uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60 # v2.0.9
```

Comment:
```suggestion
        uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60 # v2.0.10
```

Reply:
or, since dependabot isn't smart enough to update these comments, maybe we should just remove them?

Reply:
yeah, let's just remove them

Reply:
```suggestion
        uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60
```